src/core/src/op/util/broadcast_base.cpp

```diff
@@ -463,8 +466,10 @@ bool ov::op::util::BroadcastBase::evaluate(const HostTensorVector& outputs, cons
 
     if (m_mode.m_type == BroadcastType::NONE) {
         AxisVector axes_mapping_val;
-        const auto axes_mapping_constant =
-            ov::as_type_ptr<ngraph::op::v0::Constant>(input_value(2).get_node_shared_ptr());
+        std::shared_ptr<ngraph::op::v0::Constant> axes_mapping_constant = nullptr;
+        if (get_input_size() >= 3)
+            axes_mapping_constant = ov::as_type_ptr<ngraph::op::v0::Constant>(input_value(2).get_node_shared_ptr());
+
```

Comment:
it should be read from inputs

Reply:
@jane-intel do you mean we have to read `axes_mapping_constant` from HostTensor inputs in any case? (And default policy of evaluate() functions is to use HostTensors instead of node's inputs?) Cause currently if there is no `input_value(2)` we will try to read values from HostTensors at [line  #477](https://github.com/openvinotoolkit/openvino/blob/c928f7bf0ea955e50524f1bf5197ad865da94ce2/src/core/src/op/util/broadcast_base.cpp#LL477)

Reply:
We have to read from Host Tensor no matter what. This is why we pass this data. Actual inputs of original operation is irrelevant

Reply:
Oh, I see, thanks for clarification! I'll update these changes